www/artworks/post.php

```diff
@@ -139,6 +176,9 @@
 	if($httpMethod == HTTP_PATCH && $artwork !== null){
 		header('Location: ' . $artwork->Url);
 	}
+	else if($httpMethod == HTTP_PUT && $artwork !== null){
```

Comment:
After PUTting we want to redirect to the new artwork, not the edit form. Logic should be the same as PATCH

Reply:
That already works, i.e., the user does get redirected to the new artwork after a successful PUT. That happens above on line 105. This is the code that redirects the user after an error. I believe it's doing the right thing. 

(I believe the PR was confusing because Github was hiding so many unchanged lines between lines 105 and 179.)